_posts/en/newsletters/2023-05-10-newsletter.md

```diff
@@ -0,0 +1,218 @@
+    before they know for certain how much money it will earn.  For
+    example, a miner who purchases enough equipment to receive 1% of the
+    network's current total of rewards might be surprised to find that
+    other miners also purchased enough equipment to double the total
+    network hashrate, leaving the miner with 0.5% of the reward instead
+    of 1%.  With PoWSwap, the miner can make a trustless contract with
+    someone who is willing to pay the miner if hashrate increases before
+    a certain date, offsetting the miner's lower than expected income.
+    In exchange, the miner pays that person an upfront premium or agrees
+    to pay them a larger amount if network-wide hashrate stays the same
+    or decreases.
+
+  - *Hashrate decrease insurance:* a wide variety of problems with
+    bitcoin would result in a significant decrease in network-wide
+    hashrate.  Hashrate would decrease if miners were being shutdown by
+    powerful parties, or if a significant amount [fee sniping][topic fee
+    sniping] began to suddenly occur among established miners, or if the
+    value of BTC to miners suddenly decreased.  Holders of BTC who
+    wanted to insure against such situations could enter trustless
+    contracts with miners or third-parties.
+
+  - *Exchange rate contracts:* in general, if the purchasing power of
+    BTC increases, miners are willing to increase the amount of hashrate
+    they provide (to increase the rewards they receive).  If purchasing
+    power decreases, hashrate decreases.  Many people may be interested
+    in trustless contracts related to the future purchasing power of
+    Bitcoin.
+
+  Although the idea for PoWSwap has been circulated for several years,
+  the paper provides more detail and analysis than we've previously
+  seen.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+<!-- FIXME:harding to update on Tuesday -->
+
+- [Core Lightning 23.05rc2][] is a release candidate for the next
+  version of this LN implementation.
+
+- Bitcoin Core 24.1 FIXME:harding
+
+- Bitcoin Core 25.0 FIXME:harding
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:LarryRuane
+
+[Don't download witnesses for assumed-valid blocks when running in prune mode][review club 27050]
+is a PR by Niklas Gögge (dergoegge) that improves the performance of Initial Block Download
+(IBD) by not downloading witness data on nodes that are configured to both
+[prune block data][docs pruning] and use [assumevalid][docs assume valid]. This
+optimization was discussed in a recent [stack exchange question][se117057]. {% assign timestamp="43:42" %}
```

Comment:
```suggestion
optimization was discussed in a recent [Stack Exchange question][se117057]. {% assign timestamp="43:42" %}
```

Reply:
Sorry, this was pasted from a previous newsletter to provide a template for this section, but I forgot to clean it up.  I'm going to leave this as-is so it either gets replaced by Larry or deleted by me/Mike if there's no Bitcoin Core PR Review Club summary.

Reply:
>this was pasted from a previous newsletter

That is perfectly okay, I figured as much, I'll replace it. I'm making good progress but I have to run out on an errand soon and won't be back until later tonight (Denver time) so I should be able to finish this up around midnight my time (0600 UTC), will that be okay?

Reply:
That's fine.  I'll try to review it before I go to bed, otherwise @bitschmidty can make any last-minute edits.  Thanks (and sorry again), @LarryRuane!

Reply:
Sounds good, no need to apologize, I should apologize since I should have caught this myself! I now have a calendar entry for the second Wednesday of each month (with a one-week reminder) so this shouldn't happen in the future.